test/featureUtils_unit_test.js

```diff
@@ -0,0 +1,48 @@
+/* global describe, assert, it, itowns */
+require('./itowns-testing.js');
+var example = require('../examples/loadGeojson.js');
```

Comment:
Why don't you directly load `../examples/geojson/simple.geojson` from here?

And `itowns-testing.js` is definetely not adequate for unit testing. You should directly import the required modules (see `coordinate_unit_test.js`).

But besides this: 👍 for adding tests!

Reply:
updated unit test